### PR TITLE
Implement the unit test related to the multitenacy user story.

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@flopezag 

### DESCRIPTION
Implemented the unit tests related to the multitenacy user story.

### CODE REVIEW

File: tests/test_redis.py [hunk 1]
```diff
@@ -273,3 +275,47 @@ def testGetWidowSizeOfAnUnexistingTenant(self):
...
         self.assertEqual(expectedvalue, result)
+
+    def testGetWindowSize_diferent_tenants(self):
+        """test should return the window size of different tenant to check multitenacy"""
+        p = myredis()
+
+        expectedvalue1 = ["['tenantid', 4]"]
+        expectedvalue2 = ["['different_tenant_id', 5]"]
+        p.insert_window_size(tenantid, ['tenantid', 4])
+        p.insert_window_size(tenantid2, [tenantid2, 5])
```
Comment (flopezag) [inline tests/test_redis.py]: [tenantid2, 5] should work?

Comment (geonexus) [inline tests/test_redis.py]: Yes, it should work as a second tenant

Comment (flopezag) [inline tests/test_redis.py]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**